.ci/Jenkinsfile

```diff
@@ -32,7 +38,10 @@ pipeline {
         stash allowEmpty: true, name: 'source', useDefaultExcludes: false
         dir("${BASE_DIR}"){
           setEnvVar('GO_VERSION', readFile(".go-version").trim())
+          // TODO: get the version for this project
+          setEnvVar('VERSION', sh(label: 'Get version', script: 'make get-version', returnStdout: true)?.trim())
```

Comment:
TBC, what's the file or command to be executed so the version of the project can be saved in the env varaible to be used later on

Reply:
I've added https://github.com/elastic/elastic-agent-shipper/pull/102 to retrieve the version